homeassistant/components/sensor/rmvtransport.py

```diff
@@ -0,0 +1,202 @@
+"""
+Support for real-time departure information for Rhein-Main public transport.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/sensor.rmvtransport/
+"""
+import logging
+from datetime import timedelta
+
+import voluptuous as vol
+
+import homeassistant.helpers.config_validation as cv
+from homeassistant.helpers.entity import Entity
+from homeassistant.components.sensor import PLATFORM_SCHEMA
+from homeassistant.const import (CONF_NAME, ATTR_ATTRIBUTION)
+
+REQUIREMENTS = ['PyRMVtransport==0.0.7']
+
+_LOGGER = logging.getLogger(__name__)
+
+CONF_NEXT_DEPARTURE = 'next_departure'
+
+CONF_STATION = 'station'
+CONF_DESTINATIONS = 'destinations'
+CONF_DIRECTIONS = 'directions'
+CONF_LINES = 'lines'
+CONF_PRODUCTS = 'products'
+CONF_TIME_OFFSET = 'time_offset'
+CONF_MAX_JOURNEYS = 'max_journeys'
+
+DEFAULT_NAME = 'RMV Journey'
+
+VALID_PRODUCTS = ['U-Bahn', 'Tram', 'Bus', 'S', 'RB', 'RE', 'EC', 'IC', 'ICE']
+
+ICONS = {
+    'U-Bahn': 'mdi:subway',
+    'Tram': 'mdi:tram',
+    'Bus': 'mdi:bus',
+    'S': 'mdi:train',
+    'RB': 'mdi:train',
+    'RE': 'mdi:train',
+    'EC': 'mdi:train',
+    'IC': 'mdi:train',
+    'ICE': 'mdi:train',
+    'SEV': 'mdi:checkbox-blank-circle-outline',
+    None: 'mdi:clock'
+}
+ATTRIBUTION = "Data provided by opendata.rmv.de"
+
+SCAN_INTERVAL = timedelta(seconds=30)
```

Comment:
Do we need to update more frequently than once a minute? The unit of measurement is minutes.

Reply:
You're probably right. Every minute should be sufficient. 30 seconds was simply to make sure not to query just before the values actually get updated and therefore miss "the bus". Does that make sense?

Reply:
Ok, you decide.